.github/vale/dicts/aiven.dic

```diff
@@ -143,6 +143,7 @@ keystores
 Kibana
 Klaw
 Kpow
+KafkaStreams
```

Comment:
Seems the existing standard is to write `Apache Kafka® Streams`

Reply:
Thanks for finding this out. Replaced KafkaStreams with  Apache Kafka® Streams